numpy/_core/numeric.py

```diff
@@ -1016,9 +1016,10 @@ def tensordot(a, b, axes=2):
     * ``axes = 1`` : tensor dot product :math:`a\\cdot b`
     * ``axes = 2`` : (default) tensor double contraction :math:`a:b`
 
-    When `axes` is integer_like, the sequence for evaluation will be: first
-    the -Nth axis in `a` and 0th axis in `b`, and the -1th axis in `a` and
-    Nth axis in `b` last.
+    When `axes` is integer_like and positive, the sequence for operation will 
+    first start with the Nth position from the last (axis = -N) in `a` and 
+    the first position (axis = 0) in `b` and continue until the last position 
+    (axis = -1) in `a` and Nth position (axis = N-1) in `b`.
```

Comment:
Tiny grammar nit:

```suggestion
    When `axes` is integer_like and positive, the operation starts with the
    Nth position from the last (axis = -N) in `a` and the first position
    (axis = 0) in `b` and continue until the last position 
    (axis = -1) in `a` and Nth position (axis = N-1) in `b`.
```
Thanks!